src/Stripe.net/Infrastructure/FormEncoding/MimeTypes.cs

```diff
@@ -0,0 +1,45 @@
+namespace Stripe.Infrastructure.FormEncoding
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Collections.Immutable;
+
+    internal static class MimeTypes
+    {
+        private static readonly IDictionary<string, string> MimeTypeMap
```

Comment:
I realize this is basically refactored from another piece of code, but mind writing up some justification for why these particular file extensions are handled and why they're deemed to be a sufficient set for what's likely to be uploaded?

As someone not too familiar with the code, it'd be nice to know when it's appropriate to add a new supported file.

Reply:
The new dictionary contains all the mimetypes that can be used when uploading a file. We don't need to be very diligent about keeping the list up-to-date: the backend doesn't trust the `Content-Type` and determines the type from the actual file contents anyway. I considered removing this entirely and always using `application/octet-stream` for the `Content-Type` (some of the other clients do this IIRC), but I think this is a bit nicer and potentially helpful when debugging.

I'll add a comment to explain the above :)